src/main/scala/edu/ie3/simona/agent/grid/congestion/CongestedComponents.scala

```diff
@@ -87,6 +90,10 @@ object CongestedComponents {
       .filter { case (line, res) => res > line.iNom }
 
     // checking for transformer congestions
+    implicit def asApparent(power: Power): ApparentPower = Kilovoltamperes(
+      power.toKilowatts
+    )
+
```

Comment:
Ok now I see why you did this. I'm not sure if I like this more than converting explicit to Amperes and Kilovolts below and then apply KiloVoltAmperes. May @sebastian-peter has some opinion on this?

Reply:
I will replace the implicit conversion with an explicit one.